lib/utils.js

```diff
@@ -9,7 +12,7 @@
  *
  * @returns {object|boolean} - Will return the first filtered object, or `false` if no objects match
  */
-exports.singleItem = (key, value, arr) => {
+module.exports.singleItem = (key, value, arr) => {
```

Comment:
the `module.exports` in this file differentiates from the way other files are doing it.